_posts/en/2019-01-20-rbf-in-the-wild.md

```diff
@@ -0,0 +1,242 @@
+---
+title: 'RBF in the Wild'
+permalink: /en/rbf-in-the-wild/
+name: 2019-01-20-rbf-in-the-wild
+type: posts
+layout: post
+lang: en
+version: 1
+---
+
+Opt-in Replace by Fee was standardized in December 2015. But who is supporting it and what is the user experience like? This post is a study of Bitcoin wallet and block explorer support and usability of Opt-in RBF (BIP125). The findings were initially presented during the most recent [Bitcoin Optech Workshop in Paris](https://bitcoinops.org/workshops/).
+
+Understanding how other wallets, exchanges, and block explorers handle Opt-in RBF transactions is an important consideration for high-transacting Bitcoin services. In order for exchanges or wallets to support RBF, they want to know that there is broad support for RBF and that user experience in receiving wallets or block explorers is not confusing for users.
+
+The goal of this study is to evaluate how Opt-in RBF is supported in the wallet and explorer ecosystem. The study is meant to be more descriptive than prescriptive. However, we do have some general recommendations. When considering usability, each service should consider the experience and knowledge of its user base. Hopefully, the compilation of this information in one location will help Bitcoin companies make informed decisions about how to support RBF and even start some discussions about what techniques work and which do not. 
+
+Transaction replacement is important as it can enable:
+
+- Wallets and users to have finer control over transaction fees since a low fee can later be bumped
+- 'Unsticking' a stuck transaction that doesn't have enough fee to be mined
+
+Some advanced scenarios which benefit from transaction replacement include:
+
+- Combining two or more payments into a single transaction
+- Iterative Payment Batching
+- Pre-computed fee bumping
+- Offchain payment techniques like payment channels benefit from replacement
```

Comment:
I think the first two items can be aggregated: "- Combining two or more payments into a single transaction (iterative payment batching)".

Reply:
Agreed.